flink-python/docs/reference/pyflink.table/catalog.rst

```diff
@@ -250,3 +250,31 @@ A CatalogDescriptor is a template for creating a catalog instance. It closely re
     :toctree: api/
 
     CatalogDescriptor.of
+
+
+ObjectIdentifier
+----------------
+
+Identifies an object in a catalog. It allows to identify objects such as tables, views,
+function, or types in a catalog. An identifier must be fully qualified. It is the
+responsibility of the catalog manager to resolve an identifier to an object.
+
+While Path :class:`ObjectPath` is used within the same catalog, instances of this class can be
```

Comment:
I am curious what` instances of this class can be used across catalogs.` in practise. Over what scope are they unique and how does this relate to being `fully qualified`.